apps/emqx/src/emqx_message.erl

```diff
@@ -273,14 +273,20 @@ remove_header(Hdr, Msg = #message{headers = Headers}) ->
         false -> Msg
     end.
 
--spec is_expired(emqx_types:message()) -> boolean().
-is_expired(#message{
-    headers = #{properties := #{'Message-Expiry-Interval' := Interval}},
-    timestamp = CreatedAt
-}) ->
+-spec is_expired(emqx_types:message(), atom()) -> boolean().
+is_expired(
+    #message{
+        headers = #{properties := #{'Message-Expiry-Interval' := Interval}},
+        timestamp = CreatedAt
+    },
+    _
+) ->
     elapsed(CreatedAt) > timer:seconds(Interval);
-is_expired(_Msg) ->
-    false.
+is_expired(#message{timestamp = CreatedAt}, Zone) ->
```

Comment:
Nit: is it necessary to add some test cases to cover these lines? 

And, the changes logs for this PR are missing.